resources/config_defaults.rb

```diff
@@ -0,0 +1,71 @@
+property :timeout, Hash, default: { client: '10s', server: '10s', connect: '10s' }
+property :name, String, name_property: true
+property :log, String, default: 'global'
+property :mode, String, default: 'http', equal_to: %w(http tcp)
+property :balance, default: 'roundrobin', equal_to: %w(roundrobin static-rr leastconn first source uri url_param header rdp-cookie)
+property :option, Array, default: %w(httplog dontlognull redispatch)
+property :status_uri, String, default: '/haproxy-status'
```

Comment:
This should probably be `stats_uri` to match the property from the listen resource and the config option name in `haproxy`